package.json

```diff
@@ -22,7 +22,7 @@
   "dependencies": {
     "coffeescript": "^2.5.0",
     "esutils": "^2.0.2",
-    "prettier": "github:helixbass/prettier#b8818ebd"
+    "prettier": "github:helixbass/prettier#prettier-v2.1.0-dev.100-gitpkg"
```

Comment:
I used [`gitpkg`](https://github.com/ramasilveyra/gitpkg) to make the updated Prettier fork branch consumable as a Github dependency

Doing it this way made me no longer able to access non-public API's from Prettier's internals

So went through and updated/copied some utils from Prettier (most notably the comment-printing utils)